src/vm.rs

```diff
@@ -105,19 +104,24 @@ pub struct UhyveVm<VCpuType: VirtualCPU = VcpuDefault> {
 	entry_point: u64,
 	stack_address: u64,
 	pub mem: Arc<MmapMemory>,
+	pub params: Params,
+	memory_size: usize,
```

Comment:
memory_size might be useful if we move the memory initialization outside of new(...) (perhaps using a MaybeUninit - not quite sure).

Reply:
But you can access the `memory_size` via `mem` already?

Reply:
I wasn't exactly sure whether doing this would get in the way of your work, given such a way to access this variable won't be possible after the replacement of the `MmapMemory` interface in https://github.com/hermit-os/uhyve/pull/643.

There is some technical debt in https://github.com/hermit-os/uhyve/pull/725 which does the exact same thing, but, as a trade-off, I assumed that a superfluous field would've been better than increasing that technical debt.